lib/cucumber/formatter/fail_fast.rb

```diff
@@ -0,0 +1,21 @@
+require 'cucumber/formatter/io'
+require 'cucumber/formatter/console'
+
+module Cucumber
+  module Formatter
+    class FailFast
+      def initialize(configuration)
+        @configuration = configuration
+      end
+
+      def after_test_case(test_case, result)
+        Cucumber.wants_to_quit = true unless result.ok? @configuration.strict?
```

Comment:
I don't understand this line. Is it a typo? If I remove the `@configuration.strict?` bit all the tests pass.